tests/apollo/test_skvbc.py

```diff
@@ -78,6 +78,44 @@ async def test_state_transfer(self, bft_network,exchange_keys=True):
         await bft_network.force_quorum_including_replica(stale_node)
         await skvbc.assert_successful_put_get(self)
 
+    @with_trio
+    @with_bft_network(start_replica_cmd)
+    async def test_request_missing_data_from_previous_window(self, bft_network, exchange_keys=True):
+        """
+        Test that a replica succeeds to ask for missing info from the former window
+
+        1. Start all nodes and process 149 requests.
+        2. Stop one node and process 2 more requests (this will cause to the remaining
```

Comment:
Might be we can stop F Replicas, and before we bring them back, we can verify that the other 2F+C+1 have all reached a Stable Checkpoint. I think this way we will know for sure that the restarted F are catch up from the Inactive Window.